DfE.FindInformationAcademiesTrusts/Extensions/StringExtensions.cs

```diff
@@ -26,4 +27,11 @@ public static string Kebabify(this string text)
 
         return transformedText;
     }
+
+    public static string ToTitleCase(this string text)
+    {
+        var textInfo = CultureInfo.CurrentCulture.TextInfo;
```

Comment:
Add a guard clause i.e., if text = null return empty string otherwise we'll see a null ref exception

Reply:
Specifically it would throw on the .tolower()